tests/Build/Changelog/ChangelogBuilderTest.php

```diff
@@ -0,0 +1,141 @@
+<?php
+namespace Aws\Test\Build\Changelog;
+
+use Aws\Build\Changelog\ChangelogBuilder;
+
+class ChangelogBuilderTest extends \PHPUnit_Framework_TestCase
+{
+	/**
+	 * @covers ChangelogBuilder
+	 *
+	 */
+
+	private $RESOURCE_DIR= "tests/Build/Changelog/resources";
+
+	private function getChangelogBuilder()
```

Comment:
This was made so that in future if we want to change the constructor or add more variables then we will have only one place to change.